boards/xtensa/qemu_xtensa/qemu_xtensa_defconfig

```diff
@@ -10,3 +10,4 @@ CONFIG_TOOLCHAIN_VARIANT=""
 CONFIG_SW_ISR_TABLE=y
 CONFIG_GEN_ISR_TABLES=y
 CONFIG_GEN_IRQ_VECTOR_TABLE=n
+CONFIG_SIMULATOR_XTENSA=y
```

Comment:
Looking quickly, it seems like, among other bookeeping, this will enable the xtensa_sim_console.c driver, which uses simcall instructions to emit output.  Does qemu support that?  If so might be good to call that out in the commit message.  Is anything else going to break if we switch from whatever the earlier console device was to this one?

Reply:
IIRC, qemu_xtensa and xt-sim both use simcall interfaces to talk to the emulator. if this broke console messages my expectation would be that sanitycheck would complain about it since it tests qemu_xtensa.